src/components/HistoryTables/MsigProposals/Proposal.tsx

```diff
@@ -212,26 +201,18 @@ export default function ProposalDetail(props: ProposalDetailProps) {
       {seeMore && (
         <>
           <Line label='Next Transaction ID'>{stateData?.State.NextTxnID}</Line>
-          <Line label='Approvers'>
-            {proposal?.approved.map((approver: Address) => {
-              return (
-                <Line key={`${1}-${approver.robust || approver.id}`} depth={1}>
-                  {approver.robust}{' '}
-                  <a
-                    target='_blank'
-                    rel='noreferrer noopener'
-                    href={props.addressHref(approver.robust)}
-                  >
-                    ({approver.id})
-                  </a>
-                  <CopyText
-                    text={approver.robust}
-                    color='core.primary'
-                    hideCopyText={false}
-                  />
-                </Line>
-              )
-            })}
+          <Line
+            label={`Approvers${
+              proposal?.approved ? ` (${proposal?.approved.length})` : ''
+            }`}
+          >
+            {proposal?.approved.map((approver: Address, index: number) => (
+              <AddressLink
+                key={index}
```

Comment:
lets make the `key={Address.robust || Address.id}` instead of the array index

Reply:
sounds good